community/kernel/src/main/java/org/neo4j/kernel/impl/proc/ProcedureGDBFacadeSPI.java

```diff
@@ -205,6 +184,18 @@ public void shutdown()
     @Override
     public KernelTransaction beginTransaction( KernelTransaction.Type type, SecurityContext securityContext, long timeout )
```

Comment:
Should we really provide an API to create transactions with alternative security context?